### PR TITLE
Add version fuzzing to chilly spec for conda pkgs

### PR DESCRIPTION
**Pull request by jaytmiller**
Add requirements fuzzing to chilly conda env spec for conda pkgs
Add workflow_dispatch too all action workflows

### CODE REVIEW

File: .github/workflows/build-test.yml [hunk 1]
```diff
@@ -3,12 +3,10 @@ on:
    push:
       branches:
         - main
-        - test
    pull_request:
       branches:
         - main
-        - test
-
+   workflow_dispatch:
```
Comment (cslocum) [inline .github/workflows/build-test.yml:9]: Why does `workflow_dispatch` have no attributes/values/config?

Comment (jaytmiller) [reply .github/workflows/build-test.yml:9]: Dunno,  just clone code Jake wrote to turn on the "run workflow" button elsewhere.

Comment (cslocum) [reply .github/workflows/build-test.yml:9]: OK, sounds good.

**Review by cslocum — APPROVED** [review]
:+1:

**Pull request merged**